src/Utils/Helpers.spec.ts

```diff
@@ -93,41 +94,106 @@ describe('Helpers', () => {
   });
 
   describe('getExecArgs', () => {
-    it('should return current exec args', () => {
-      const result = Helpers.getExecArgs();
+    describe('unescaped', () => {
```

Comment:
Please follow the naming convention:

```ts
describe('SUT, e.g. Helpers', () => {
  describe('class or instance method, e.g getExecArgs', () => {
    it.todo('should do something');
  });
})
```

Avoid using suites to group by input data. So, you should remove `describe('unescaped')` and `describe('escaped')` unwrapping the tests.